cli_tools_e2e_test/common/logging/logging.go

```diff
@@ -0,0 +1,69 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package logging
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"log"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+// AsDaisyLogger returns a daisy.Logger that writes to a log.Logger.
+func AsDaisyLogger(logger *log.Logger) daisy.Logger {
```

Comment:
This is kind of minor right now as it's used in tests only, but, just looking at the method/package names, it's not clear that this returns a logger that writes to log.Logger, since it's in logging package (that's not related to log package). If it was log.AsDaisyLogger() that would be clear.

Reply:
I'd agree -- this isn't intended to be a battle-tested API that's usable throughout the codebase, but rather a pragmatic set of shims that translate between a few different logging and IO interfaces. Regarding the package name `logging`, I hadn't intended for it to refer to any particular logger, but rather to logging in general.